api/app/tests/apps/users/factories.py

```diff
@@ -40,3 +41,22 @@ class SuperUserFactory(UserFactory):
 
     is_superuser = True
     is_staff = True
+
+
+class GroupFactory(factory.DjangoModelFactory):
+    class Meta:
+        model = Group
+        django_get_or_create = ('name',)
+
+    name = factory.LazyAttribute(
+        lambda o: fake.word()
+    )
+
+    @factory.post_generation
```

Comment:
Remove, new ticket in JIRA for clean-up of factories.

Reply:
Done